client/app/views/steps/password.coffee

```diff
@@ -8,15 +10,22 @@ module.exports = class PasswordView extends StepView
     events:
         'click .next': 'onSubmit'
         'click [action=password-visibility]': 'onToggleVisibility'
+        'keyup input': 'checkPasswordStrength'
 
 
     isVisible: false
 
 
-    onRender: ->
+    onRender: (args...) ->
+        super args...
         @$inputPassword = @$('input[name=password]')
         @$visibilityButton = @$('[action=password-visibility]')
         @$visibilityIcon = @$('.icon use')
+        @$errorContainer=@$('.errors')
+        if @error
+            @handleError(@error)
+        else
+            @$errorContainer.hide()
```

Comment:
If I use the condition in the template instead of here, I would be unable to show an error message directly in the view (because the error container wouldn't be generate in the view).